client/components/theme-tier/theme-tier-badge/index.js

```diff
@@ -71,6 +70,7 @@ export default function ThemeTierBadge( {
 			);
 		}
 
+		// All partner themes are also marketplace themes... Do we need this OR?
 		if ( themeTierSlug === 'partner' || themeType === MARKETPLACE_THEME ) {
```

Comment:
That's a good question. @jacopo I know it's been a long time, but perhaps you still remember the why behind it, or have the context about it?